### ISSUE TITLE
Hide description column

### ISSUE THREAD

**Issue by xaimaran** [a85/POSTMan-Chrome-Extension#11455]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

It's very annoying in the Postman to keep hiding the description column in the Params and Headers sections.

Found nothing in searching the documentation, config files, and settings.

### Describe the solution you'd like

Expect to set it one time.

**Comment by giacomofm** [comment]
I also find that having a general setting to hide the description column can be extremely useful for having a cleaner interface. I rarely share my collection with others, so I don't use the description column much. Having to hide it every time I open the app becomes annoying.

**Comment by ClaudioCrespo** [comment]
Bump

**Comment by Dhembla** [comment]
Hey Folks,
Closing this issue as it's a duplicate of [#10605](https://github.com/postmanlabs/postman-app-support/issues/10605) issue. Also, We have added this capability as part of our latest release, v10.9, and from now on, you will be able to hide the description column and the selection will persist across sessions. The release is being rolled out and will be available in the next few days.
Thanks for your patience 🙇
